cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerLoggingConfigurationDiffTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.operator.resource;
+
+import io.strimzi.test.TestUtils;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.ConfigEntry;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class KafkaBrokerLoggingConfigurationDiffTest {
+
+    private final int brokerId = 0;
+
+    private String getDesiredConfiguration(List<ConfigEntry> additional) {
+        try (InputStream is = getClass().getClassLoader().getResourceAsStream("desired-kafka-broker-logging.conf")) {
```

Comment:
Is it even worth putting a two line file in a resource like that? Just use a String literal.

If you must use a file, the name `desired-kafka-broker-logging.conf` is a bit weird. It's a properties file, so call it with `.properties` suffix at least.